### ISSUE TITLE
The java script link doesn't have a meaningful description in Signposts tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1931]
URL: http://clarity-ks.surge.sh/signposts

Tools: Web Accessibility Toolbar (WAT2.0) 

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Signposts tab.
3. Click help-info button.
4. Check the java script link in Signposts tab.

Actual Results:
The java script link has simple description “link”, and the description is not meaningful.  
For more details, please refer to “Signposts-JavascriptLinks-NotMeaningful.png”.

Expected Results:
The java script link should have unique and meaningful description.  
Add the full description to the link.

Note:
This issue also occurs in Tabs tab.

Violation of Section 508 Standards: 
Section 1194.22(I) Frames shall be titled with text that facilitates frame identification and navigation. 
Section 1194.31(a) At least one mode of operation and information retrieval that does not require user vision shall be provided, or support for Assistive Technology used by people who are blind or visually impaired shall be provided. 
Section 1194.31(b) At least one mode of operation and information retrieval that does not require visual acuity greater than 20/70 shall be provided in audio and enlarged print output working together or independently, or support for Assistive Technology used by people who are visually impaired shall be provided. 
User Impact: Blind.

Screen shot:
1.	Signposts-JavascriptLinks-NotMeaningful.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/77277684-4cc5-4e7f-ba53-a516cefcc601)

**Comment by Jinnie** [comment]
Done. Will commit with several related A11Y issues.
